wordpress-mobile/AztecEditor-iOS
# UIPasteboard: NSCoding conformance for Aztec Attributes

### Details:
- Neutralizes PR #627 (Band Aid Fix)
- Implements NSCoding conformance in a few classes: **HTMLRepresentation** + **HTMLElementRepresentation** + **UnsupportedHTML** + **Attribute**
- Serialization capabilities for **Value** + **CSSProperty**
- Nukes **MarkerNode** (dead class!)

Fixes #626

### To test:
- Verify that the unit tests are green
- Try copy / cut + paste of literally anything!

cc @diegoreymendez @SergioEstevao 

Thanks in advance gentlemen!!
